bot.py

```diff
@@ -50,55 +64,41 @@ def get_confirm_message(lang: str):
     return CONFIRMS[lang][key]
 
 
+def detect_language(text: str) -> Optional[str]:
+    language = None
+    for name in AGREE_NAMES.keys():
+        if name in text:
+            language = AGREE_NAMES[name]
+    return language
+
+
+def check_is_reply_needed(text: str) -> bool:
+    is_reply_needed = False
+    for owl_name in OWL_NAMES.keys():
+        if owl_name in text:
+            is_reply_needed = True
```

Comment:
Same here. If we found needed result no need to finish the loop.

Reply:
Also updated